src/main/java/Storage.java

```diff
@@ -0,0 +1,155 @@
+import java.io.*;
```

Comment:
Better to avoid import of packages using *. I'm guessing you meant `import java.io.IOException;` perhaps...